ui/src/app.js

```diff
@@ -159,12 +159,12 @@ function processData (data) {
       break
     case 'wifiConnecting':
       t('wifi-connecting',
-        locale.translate('This could take a few moments.').fetch())
+        translate('This could take a few moments.'))
```

Comment:
This can become a oneliner, translate alone on the second line is a bit confusing with the extra parenthesis